### ISSUE TITLE
FillValue attribute error

### ISSUE THREAD

**Issue by freecodegaku** [cerfacs-globc/icclim#66]
Running icclim using python3.7 with the latest netcd4 and cftime packages installed produces the following:
```
    icclim.indice(indice_name='RX5day', in_files=files, var_name='RAINTOT', time_range=[dt1, dt2], slice_mode='month', out_file=out_f)
  File "icclim.py", line 216, in indice
    indice_dim = util_nc.copy_var_dim(inc, onc, var_name[0], lev_dim_pos=lev_dim_pos) # tuple ('time', 'lat', 'lon')    
  File "icclim/util/util_nc.py", line 274, in copy_var_dim
    onc_dim0.__setattr__(attr[0], attr[1])
  File "netCDF4/_netCDF4.pyx", line 4311, in netCDF4._netCDF4.Variable.__setattr__
AttributeError: _FillValue attribute must be set when variable is created (using fill_value keyword to createVariable)
```
The accompanying ncdump from the file that we are using 
```
netcdf daily_RAINTOT_d01 {
dimensions:
	time = UNLIMITED ; // (31 currently)
	lat = 152 ;
	lon = 223 ;
variables:
	float RAINTOT(time, lat, lon) ;
		RAINTOT:_FillValue = 9.96921e+36f ;
	double time(time) ;
		time:calendar = "standard" ;
		time:units = "days since 1900-01-01 00:00:00" ;
		time:_FillValue = 9.96920996838687e+36 ;
	float lat(lat, lon) ;
		lat:FieldType = 104 ;
		lat:MemoryOrder = "XY " ;
		lat:description = "LATITUDE, SOUTH IS NEGATIVE" ;
		lat:units = "degree_north" ;
		lat:stagger = "" ;
		lat:coordinates = "XLONG XLAT" ;
	float lon(lat, lon) ;
		lon:FieldType = 104 ;
		lon:MemoryOrder = "XY " ;
		lon:description = "LONGITUDE, WEST IS NEGATIVE" ;
		lon:units = "degree_east" ;
		lon:stagger = "" ;
		lon:coordinates = "XLONG XLAT" ;
}
```

note: I had to update all instances of netcdftime to cftime.

**Comment by pagecp** [comment]
Sorry for my late reply, as I am overloaded with extra work within those past weeks.
Would you have available a copy of your file?

**Comment by pagecp** [comment]
@freecodegaku I am about to close the issue but it would be great if you would send me a copy of your file for validating? Thanks!

**Comment by freecodegaku** [comment]
I have just seen both your messages. I don't the file on hand. I will try and generate some data and send that through to you.

**Comment by pagecp** [comment]
Thanks very much! 
 
I will investigate as soon as I receive your data.

**Comment by freecodegaku** [comment]
Here is some sample data.

https://cloudstor.aarnet.edu.au/plus/s/hGggU280sLpnmQf

**Comment by pagecp** [comment]
Finally I have time to work again on icclim.... the file you have provided me does not contains the lat and lon variable so I cannot test:
```
ncdump -h daily_RAINTOT_d01.nc 
netcdf daily_RAINTOT_d01 {
dimensions:
	time = UNLIMITED ; // (182 currently)
	lat = 152 ;
	lon = 223 ;
variables:
	float RAINTOT(time, lat, lon) ;
		RAINTOT:_FillValue = 9.96921e+36f ;
	double time(time) ;
		time:calendar = "standard" ;
		time:units = "days since 1900-01-01 00:00:00" ;
		time:_FillValue = 9.96920996838687e+36 ;
}
```

**Comment by bzah** [comment]
Closing this as it is most likely fixed with icclim 5.0.0.
Now, we let xarray open the netcdf and most of the time we don't handle _fillValue directly in icclim.
